sphinx/domains/c.py

```diff
@@ -3552,7 +3552,6 @@ def _render_symbol(self, s: Symbol, maxdepth: int, skipThis: bool,
 
     def apply(self, **kwargs: Any) -> None:
         for node in self.document.findall(AliasNode):
-            node = cast(AliasNode, node)
```

Comment:
Good point. Our stubs surely provide nice annotation for `findall()`!
https://github.com/tk0miya/docutils-stubs/blob/b14071d065f8ee35399856fcaff6cd608294a342/docutils-stubs/nodes.pyi#L32-L33